ci-operator/step-registry/windows/e2e/operator/test/mirror-images/windows-e2e-operator-test-mirror-images-commands.sh

```diff
@@ -53,10 +53,10 @@ do
  --skip-missing=true --skip-verification=true --keep-manifest-list=true --filter-by-os='.*'
 
     source_image=$(echo "$image" | cut -d'=' -f1)
-    mirror_placeholder=$(echo "$image" | cut -d'=' -f2)
+    mirror_registry=$(echo "$image" | cut -d'=' -f2)
 
     itms_content+="  - source: $source_image\n"
-    itms_content+="  destination: $mirror_placeholder\n"
+    itms_content+="  destination: $mirror_registry\n"
```

Comment:
please test this logic locally as I showed on slack. this does not generate a valid ITMS
here's my attempt:

```
itms_content="apiVersion: config.openshift.io/v1\n"
itms_content+="kind: ImageTagMirrorSet\n"
itms_content+="metadata:\n"
itms_content+="  name: wmco-e2e-mirrorset\n"
itms_content+="spec:\n"
itms_content+="  imageTagMirrors:\n"

for image in $(cat /tmp/mirror-images-list.yaml)
do
   oc image mirror $image  --insecure=true -a "${new_pull_secret}" \
 --skip-missing=true --skip-verification=true --keep-manifest-list=true --filter-by-os='.*'

    source_image=$(echo "$image" | cut -d'=' -f1)
    mirror_registry=$(echo "$image" | cut -d'=' -f2)

    # Remove the tag and its preceding colon
    # e.g. mcr.microsoft.com/powershell:lts-nanoserver-ltsc2022 turns into mcr.microsoft.com/powershell
    source_tag_removed="${source_image%:*}"
    mirror_tag_removed="${mirror_registry%:*}"

    itms_content+="  - source: $source_tag_removed\n"
    itms_content+="    mirrors:\n"
    itms_content+="    - $mirror_tag_removed\n"
done
```